src/js/menu/menu-button.js

```diff
@@ -29,6 +29,14 @@ class MenuButton extends ClickableComponent {
 
     this.update();
 
+    /**
+     * Hide the menu if the number of items is less than or equal to this threshold. This defaults
+     * to 0 and whenever we add items to the menu we'll increment it.
```

Comment:
If I'm going to be pedantic, this ought to be "... and whenever we add items *which can be hidden* to the menu ..." I don't want future additions to the code to increment this when they add non-hideable items!